packages/marketplace/src/constants/routes.ts

```diff
@@ -1,4 +1,6 @@
 const Routes = {
+  Authentication: '/authentication',
+  AuthenticationLoginType: '/authentication/:loginType',
   CLIENT: '/client/apps',
   CLIENT_APP_DETAIL: '/client/apps/:appid',
```

Comment:
should be uppercase following convention

Reply:
I missed this, yes, should be uppercase